docs/authmodel.md

```diff
@@ -0,0 +1,58 @@
+
+There is a special type of Udaru users, named SuperUsers, that belong to the 'ROOT' organization. SuperUsers can be assigned full access rights on all actions on all resources. To be able to do operations as SuperUser on an Organization, Udaru provides an impersonation functionality: by passing in the Http headers an 'org' field, the endpoint request is made as if the SuperUser belongs to the 'org' organization thus having acess to all the exposed Udaru functionality for that organization (like user management, team management).
+
+curl impersonation example in which a SuperUser impersonates an user belonging to the `WONKA` organization:
+```javascript
+curl -X PUT --header 'Content-Type: application/json' --header 'Accept: application/json' --header 'authorization: ROOTID' --header 'org: WONKA' -d '{"policies":["PolicyID"]}' 'http://localhost:8080/authorization/teams/TeamID/policies'
+```
+
+### Needed model structure
+
+The structure to be modeled is the following:
+
+We have the default root organization on which we have two Teams. On the first team there are two users, on the 2nd team there is one user and the 4th user belongs to no team. All the 4 users belong to the root organization.
+We plan to add to Team1 a set of policies so ithat the User1 and User2 have access to resources from Org1 and Org2, then assign to Team2 a set of policies so that User3 has access to Org3. User4 has no policies attached and has no access to any of the Organizations.
+
+A visual representation of the structure:
+```
+  -------------------   --------   --------   --------
+  |     rootOrg     |   | Org1 |   | Org2 |   | Org3 |
+  -------------------   --------   --------   --------
+     /        \   \
+   Team1     Team2 \
+   /   \       |    \
+ User1 User2 User3  User4
+```
+
+### Solution
+
+One **limitation** of this modelling approach is that we have to build the route access policies and also the resource access policies in the root organization so that they can be attached to the SuperUser teams. The policy management is made at the root organization level.
+
+The structure is build like it is described in the previous section.
+The organization endpoints access policies and organization resource access policies are attached to the two teams.
+
+A fully working model sample can be seen in the [Full organization test file][] in the "SuperUsers with limited acces across organizations" Experiment.
+
+The policies built to configure access structure are of three types:
+- Access to organization management operations is given by attaching to the two root teams the default organization policies: `authorization:organizations:read`, `authorization:teams:*`, `authorization:users:*`, `authorization:policies:list`, `authorization:policies:read`,
```

Comment:
i changed here to describe the actions and endpoints in which they give rights, the internal resource representation is less important for external users.